nixos/modules/services/networking/consul.nix

```diff
@@ -99,6 +99,7 @@ in
 
       extraConfig = mkOption {
         default = { };
+        type = types.attrsOf types.str;
```

Comment:
I tried running the consul NixOS test locally and it failed cuz not everything is a `types.str`.

```suggestion
        type = types.attrsOf types.anything;
```